sbt-exercise/src/sbt-test/sbt-exercise/basic/project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=1.2.8
```

Comment:
In the scripted  tests, the `build.properties` properties is not usually included to be aligned with the outer sbt version